components/Chat/Message/MessageReactionsList.tsx

```diff
@@ -6,13 +6,9 @@ import {
   useProfilesStore,
 } from "@data/store/accountsStore";
 import { useSelect } from "@data/store/storeHelpers";
-import {
-  backgroundColor,
-  textPrimaryColor,
-  textSecondaryColor,
-  tertiaryBackgroundColor,
-} from "@styles/colors";
-import { AvatarSizes, BorderRadius, Paddings, Margins } from "@styles/sizes";
+import { backgroundColor, textSecondaryColor } from "@styles/colors";
+import { AvatarSizes, BorderRadius } from "@styles/sizes";
```

Comment:
While working on the reactions do you think you could remove the deprecated stuff? 😮 

If it's out of the scope no worries